### DISCUSSION TITLE
issue when running jest test with scss import

### DISCUSSION THREAD

**Discussion by hieunm22** [katspaugh/wavesurfer.js#3592] [Q&A]
I use a component which import wavesurfer to display audio as wave, version 7
everything works fine until I add scss file in import section
```typescript
import "./myscssfile.scss"  // importing this file makes the test run failed
```

`
Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.
`

Run the component with scss import with debug mode doesn't have any error, just fail when running test only

I don't know whether this issue comes from jest or wavesurfer so I post this question here to get help

**Comment by katspaugh** [comment]
I don't see how this is related to wavesurfer. You imported some unrelated module that broke your tests, so the problem is probably with that module/your jest setup.
